manifests/b/byCrookie/TypeCode/1.1.9/byCrookie.TypeCode.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: byCrookie.TypeCode
+PackageVersion: 1.1.9
+InstallerType: msi
+InstallModes:
+  - silent
```

Comment:
```suggestion
  - silent
AppsAndFeaturesEntries:
  - DisplayVersion: 1.0.0.0
```

Reply:
Thank you for the display version hint. I have updated the msi installer to include the display version.